entropy.cabal

```diff
@@ -69,7 +69,7 @@ library
       cc-options:  -DisWindows
       extra-libraries: advapi32
     else
-      Build-Depends: unix
+      Build-Depends: unix >= 2.8
```

Comment:
This is quite restrictive requirement. CPP would be better for many install plans.

Reply:
Agreed.  Fixed.